src/components/App.jsx

```diff
@@ -0,0 +1,181 @@
+import React, { Component } from 'react'
+import $ from 'jquery'
+import Snackbar from 'material-ui/lib/snackbar'
+
+
+// Components
+import PhotoGallery from './PhotoGallery.jsx'
+import NavBar from './NavBar.jsx'
+
+export default class App extends Component {
+ constructor() {
+   super()
+   this.state = {
+     photos: [],
+     editable: true,
+     deleteItemModalOpen: false,
+     deleteItemsModalOpen: false,
+     showAlert: false,
+     alertMessage: '',
+     deleteItemId: 0
+   }
+ }
+
+ componentDidMount() {
+   $.ajax({
+     url:'https://pixabay.com/api/?key=2413754-c74e1a137dd08e0804625304c&q=yellow+flowers&image_type=photo',
+     dataType: 'json',
+     cache: false,
+     success: (response) => {
+       let photos =  response.hits
+       photos.filter((photo) => {
+         photo.checked = false
+       })
+       this.setState({ photos })
+     },
+     error: (xhr, status, err) => {
+       console.log(err)
+     }
+   })
+ }
+
+ checkItem(index, checked) {
+  let photos = this.state.photos
+
+  let photo = Object.assign({}, photos[index], {
+    checked: !checked
+  })
+
+  photos[index] = photo
+
+  this.setState({ photos })
+ }
+
+ handleDelete(deleteItemId) {
+   this.setState({ deleteItemId })
+   this.openModal('deleteItem')
+ }
+
+ deleteItem() {
+    let photos = this.state.photos.filter((photo) => {
+      return this.state.deleteItemId !== photo.id
+    })
+
+    this.showAlert('Image has been deleted')
+
+    this.setState({
+      photos,
+      deleteItemModalOpen: false
+    })
+  }
+
+  handleDeleteItems() {
+    let itemIsChecked
+
+    this.state.photos.filter((photo) => {
+      if(photo.checked) {
+        itemIsChecked = true
+      }
+    })
+
+    if(itemIsChecked) {
+      return this.setState({ deleteItemsModalOpen: true })
+    } else {
+      return this.showAlert('No item(s) have been selected!')
+    }
+  }
+
+  deleteItems() {
+    let photos = this.state.photos.filter((photo) => {
+      return photo.checked !== true
+    })
+
+    this.showAlert('Image(s) have been deleted')
+
+    this.setState({
+      photos,
+      deleteItemsModalOpen: false
+    })
+  }
```

Comment:
The naming of these functions is a bit hard to parse (what's the difference between `handleDelete` and `deleteItem`? Either different names or some comments in the code would help.

Also, is there any way you could reduce some duplicate code by combining the single and multiple delete functions?